src/app/modules/shared_models/user_item.nim

```diff
@@ -18,6 +19,17 @@ type
     Trusted
     Untrustworthy
 
+proc toVerificationRequestStatus*(value: VerificationStatus): VerificationRequestStatus =
```

Comment:
Is this a leftover from a previous commit?

Reply:
It's used here:
https://github.com/status-im/status-desktop/blob/19a8cb2fd3674f1d2fa676f6162164a857f68551/src/app/modules/main/profile_section/contacts/module.nim#L279
https://github.com/status-im/status-desktop/blob/19a8cb2fd3674f1d2fa676f6162164a857f68551/src/app/modules/main/profile_section/contacts/module.nim#L94

This is to prevent warning about direct enum-enum conversion. Or smth like that, I don't really remember what it was already 😄